packages/expo-av/android/build.gradle

```diff
@@ -56,7 +59,35 @@ android {
     targetSdkVersion safeExtGet("targetSdkVersion", 30)
     versionCode 24
     versionName "10.2.0"
+
+    externalNativeBuild {
+      cmake {
+        cppFlags "-O2 -frtti -fexceptions -Wall -fstack-protector-all"
+        abiFilters 'x86', 'x86_64', 'armeabi-v7a', 'arm64-v8a'
+        arguments "-DANDROID_STL=c++_shared",
+            "-DREACT_NATIVE_DIR=${REACT_NATIVE_DIR}"
+      }
+    }
   }
+
+  externalNativeBuild {
+    cmake {
+      path "CMakeLists.txt"
+    }
+  }
+
+  buildFeatures {
+    prefab true
+  }
+
+  packagingOptions {
+    excludes = ["**/libc++_shared.so"]
```

Comment:
maybe there're more? like this: https://github.com/expo/expo/blob/bc99dc184a72788f301932b1f6730d5fcb5450ac/android/expoview/build.gradle#L167-L178